docs/install/install-node.rst

```diff
@@ -4,166 +4,99 @@
-     * To enable Flocker agents to communicate with the control service and for external access to the API, add a custom TCP security rule enabling access to ports 4523-4524.
-     * Keep in mind that (quite reasonably) the default security settings firewall off all ports other than SSH.
-     * For example, if you run the MongoDB tutorial you won't be able to access MongoDB over the Internet, nor will other nodes in the cluster.
-     * You can choose to expose these ports but keep in mind the consequences of exposing unsecured services to the Internet.
-     * Links between nodes will also use public ports but you can configure the AWS VPC to allow network connections between nodes and disallow them from the Internet.
-     * If you run the MongoDB tutorial using AWS, you will need to open port 27017 to allow your MongoDB client to connect to the database.
-
-   * Review to ensure your instances have sufficient storage and your security groups have the required ports.
-
-   Launch when you are ready to proceed.
-
-#. Add the *Key* to your local key chain (download it from the AWS web interface first if necessary):
-
-   .. prompt:: bash alice@mercury:~$
-
-      mv ~/Downloads/my-instance.pem ~/.ssh/
-      chmod 600 ~/.ssh/my-instance.pem
-      ssh-add ~/.ssh/my-instance.pem
-
-#. Look up the public DNS name or public IP address of each new instance.
-   Log in as user ``centos`` (or the relevant user if you are using another AMI).
-   For example:
-
-   .. prompt:: bash alice@mercury:~$
-
-      ssh centos@ec2-AA-BB-CC-DD.eu-west-1.compute.amazonaws.com
-
-#. Allow SSH access for the ``root`` user on each node, then log out.
-
-   .. task:: install_ssh_key
-      :prompt: [user@aws]$
-
-#. Log back into the instances as user "root" on each node.
-   For example:
-
-   .. prompt:: bash alice@mercury:~$
-
-      ssh root@ec2-AA-BB-CC-DD.eu-west-1.compute.amazonaws.com
-
-
-#. Follow the operating system specific installation instructions below on each node.
-
-.. _rackspace-install:
-
-Using Rackspace
-===============
-
-Another way to get a Flocker cluster running is to use Rackspace.
-You'll probably want to setup at least two nodes.
+.. _centos-7-install:
 
-#. Create a new cloud server:
+Installing on CentOS 7
+======================
 
-   * Visit https://mycloud.rackspace.com
-   * Click "Create Server".
-   * Choose a supported Linux distribution (either CentOS 7 or Ubuntu 14.04) as your image.
-   * Choose a Flavor.
-     We recommend at least "8 GB General Purpose v1".
-   * Add your SSH key
+.. note:: You should ensure your nodes are Flocker-ready, either by checking the :ref:`prerequisites<installing-flocker-node-prereq>` above, or by following our guides on using :ref:`AWS<aws-install>` or :ref:`Rackspace<rackspace-install>`.
 
-#. SSH in:
+#. **Log into the first node as root.**
```

Comment:
:question: I prefer list headings like this to end with a colon, as the content is a description of how to achieve the title.